day2/part2.py

```diff
@@ -0,0 +1,26 @@
+import re
+import copy
+
+def multiplyList(myList):
```

Comment:
Instead of your own function you can use something like math.prod on the values of the dict.
But good job. 👍